app/models/application_record.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  include ActiveStorageSupport::SupportForBase64
```

Comment:
In how many models are you using this? Do you think that it's necessary for all models to include this support?

Reply:
Right. I'll move this to User model.